src/types/constraints.rs

```diff
@@ -384,8 +389,11 @@ impl<T: Default + Copy> Bounded<T> {
         }
     }
 
-    pub fn minimum(&self) -> T {
-        self.as_minimum().copied().unwrap_or_default()
+    pub const fn minimum(&self) -> T {
+        match self.as_minimum() {
+            Some(value) => *value,
+            None => T::ZERO,
```

Comment:
Isn't the minimum for signed integers below zero?

Reply:
It follows the existing default behavior, e.g. `isize` default is 0. I am not however sure should it be actually different.